test/test-app/app.module.ts

```diff
@@ -10,7 +13,14 @@ import { AppController } from './app.controller';
       excludeReqPath: '/health',
       shortBodyLength: 100,
     }),
+    GraphQLModule.forRoot<ApolloDriverConfig>({
```

Comment:
I was having a question on `what is the best place to define and import the GraplqlModule?` I was defining it in the CommonModule like what we do for `CouchbaseModule`, and then import this Module in the other modules that need it. Should we do the same here instead of directly define its initialization on the AppModule?

Reply:
we can move the discussion to the original issue https://github.com/Wiredcraft/nestjs-boilerplate/issues/192
as a test setup, I prefer to keep it slim.